01-variability.Rmd

```diff
@@ -34,124 +33,124 @@ Within the context of climate model ensembles, internal variability refers to th
+In addition, the Reliability Ensemble Average (REA) approach was developed to assess and weight the reliability of the individual models. In this method, the projections are assessed on the basis of their agreement with observed climate data and their convergence with each other, which enables a reliable estimate of future climate conditions. REA is particularly useful for regionalised climate modelling, as different models may have different strengths and weaknesses in different regions (@smith2009). 
+In addition, multivariate analysis methods play a decisive role in the creation of climate model ensembles. These approaches make it possible to consider several climate variables simultaneously and to analyse the interactions between them. Such a method improves the accuracy and reliability of the projections by better accounting for the inherent uncertainties of the models (@smith2009). 
+The cross-validation method is another important tool for validating and fine-tuning climate model ensembles. This involves comparing the predictions of the models with new, independent data sets to check the accuracy and assumptions of the statistical models. This method ensures that the climate models not only accurately reproduce the historical data, but can also provide reliable forecasts for the future (@smith2009). 
 Finally, the consideration of internal climate variability is of central importance for the creation of climate ensembles. This variability, which is characterised by both atmospheric and oceanic circulation patterns, contributes substantial to the uncertainty in climate projections. Methods such as ensemble simulations, which run a large number of simulations with a single model, make it possible to capture both the forced response of the climate system and the internal variability (@deser). 
 Overall, these methods represent different approaches to reducing uncertainties and improving the reliability of climate model ensembles. They form the basis for sound climate science findings and enable decision-makers to take more effective measures to adapt to future climate change.
 
 ## 2.3 Example of a climate model ensemble
 
-```{r EnsembleSenta, echo=FALSE, fig.align="center", fig.cap="CCSM3 40-member ensemble (@deser)", out.width="80%"}
+```{r EnsembleSenta, cache=FALSE, out.width="500", fig.align="center", echo=FALSE, eval = TRUE}
 knitr::include_graphics('work/01-variability/figures/EnsembleSenta.png')
 ```
+**Figure 1:** <font size="2">CCSM3 40-member ensemble (@deser) \
+
```

Comment:
die referenz funktioniert nur im HTML schau bitte mal unter ankündigungen in moodle da habe ich vor ein paar tagen eine lösung rein die für beides funktioniert.